app/models/owner.rb

```diff
@@ -50,8 +51,7 @@ def blocked?
 
   def affiliate_logo
     return unless affiliate
-    
-    Affiliate.find_by(code: affiliate)&.logo_url
+    affiliate.logo_url
```

Comment:
```suggestion
    affiliate&.logo_url
```
In case you have not seen before, it's called the safe navigation operator and makes sure we do not get an exception when affiliate is nil here :)